optimum/exporters/ipex/model_patcher.py

```diff
@@ -62,18 +66,22 @@ def patch_op(m, target_m, new_op_name, new_op):
 
 
 def _patch_llama_model(model):
-    if is_ipex_version("<", "2.5.0"):
-        raise ImportError("Only ipex version > 2.3.0 supports RotaryEmbedding and IndirectAccessKVCache")
+    if is_ipex_version("<", "2.3.0"):
```

Comment:
could you add a variable for this so that we can use it everywhere (also in tests)
```python
_IPEX_MINIMUM_VERSION_FOR_PATCHING = "2.3.0"
```